lib/_http_incoming.js

```diff
@@ -55,7 +55,7 @@ function IncomingMessage(socket) {
 
   if (socket) {
     streamOptions = {
-      highWaterMark: socket.readableHighWaterMark,
+      __proto__: null, highWaterMark: socket.readableHighWaterMark,
```

Comment:
```suggestion
      __proto__: null,
      highWaterMark: socket.readableHighWaterMark,
```

Reply:
fixed the rule to add multiline when needed